device-types/HPE/ProLiant-DL360e-Gen8.yaml

```diff
@@ -0,0 +1,42 @@
+---
+manufacturer: HPE
+model: ProLiant DL360e Gen8
+slug: hp_dl360e_g8
+u_height: 1.0
+is_full_depth: true
+part_number: 661189-B21
+comments: '[HP ProLiant DL360e Gen8 QuickSpecs](https://www.hpe.com/psnow/doc/c04284501)'
+console-ports:
```

Comment:
Do you have any info on the `weight` and `airflow`? If so, please add that

Reply:
Added a weight based on the maximum set(all hard drives, power supplies, and processors installed)
```
weight:
 - value: 17.92
   unit: kg
````